src/common/src/system_param/mod.rs

```diff
@@ -186,10 +173,9 @@ macro_rules! impl_system_params_from_kv {
                         std::str::from_utf8(v.as_ref()).unwrap().to_string()
                     )
                 }).collect::<Vec<_>>();
-                Err(format!("unrecognized system params {:?}", unrecognized_params))
-            } else {
-                Ok(ret)
+                tracing::warn!("unrecognized system params {:?}", unrecognized_params);
```

Comment:
Is it really safe to just ignore it? 🥵 
https://github.com/risingwavelabs/risingwave/issues/11382

Reply:
Taking `telemetry_enabled` as an example, it seems that we cannot do anything better than ignoring.

The only difference is whether to differentiate **unknown** parameters and **deprecated** parameters, such as `panic` for the former one and `warn` for the later one. But I don't think it makes any sense.

Reply:
See also https://github.com/risingwavelabs/risingwave/pull/12156#discussion_r1319666704

Reply:
All right, let's ignore it.